README.md

```diff
@@ -509,6 +509,29 @@ If for some reason you don't want the connection lazy you can turn it off by set
 ],
 ```
 
+### Network Protocol
+
+By default, the network protocol used for connection is tcp.
+If for some reason you want to use another network protocol, you can add the extra value in your config options.
+
+```php
+'connections' => [
+    // ...
+
+    'rabbitmq' => [
+        // ...
+
+        'options' => [
+            // ...
+
+            'network_protocol' => 'network wanted',
```

Comment:
I rather see this one level up. And accessed via

```php
Arr::get($config, 'network_protocol')
```

Because network connection config is defined next to `lazy` and `secure` config parameters.

The `options` are more intended for config in the worker Class.
> Now this is a bit misleading, as whe have to handle legacy config structure previous to v13.3.0

Reply:
And maybe document the available protocols

Reply:
Thanks for your review, I updated the PR with your suggestions